### DISCUSSION TITLE
Debug the content of the BlazorWebView control through Chrome DevTools Protocol

### DISCUSSION THREAD

**Discussion by stachu99** [dotnet/maui#12674] [General]
Hi.
I'd like to test a blazor content in WebView2 through Chrome DevTools Protocol (DCP) using PlayWright.
I understand that BlazowWebView uses WebView2 and is possible to connect throw CDP.
I am able to open Edge DevTools when a MAUI Hybrid app is running so I think that CDP should work.
Using Playwright with CDP connection I get 4 Pages:
1. Page Microsoft.Playwright.Core.Page file:///C:/Program%20Files/Microsoft%20Visual%20Studio/2022/Preview/Common7/IDE/CommonExtensions/Platform/CPUSampling/EdgeWebViews/detailsView.html
2. Page Microsoft.Playwright.Core.Page file:///C:/program%20files/microsoft%20visual%20studio/2022/preview/common7/ide/CommonExtensions/Platform/DiagnosticsHub/EdgeWebViews/performanceDebuggerSwimLanesView.html
3. Page Microsoft.Playwright.Core.Page file:///C:/Program%20Files/Microsoft%20Visual%20Studio/2022/Preview/Common7/IDE/PrivateAssemblies/ScriptedHost.Blank.html
4. Page Microsoft.Playwright.Core.Page file:///C:/program%20files/microsoft%20visual%20studio/2022/preview/common7/ide/CommonExtensions/Platform/DiagnosticsHub/EdgeWebViews/summaryView.html

There is not the Maui Hybrid app no matter the application is running or not.
Could you confirm that BlazorWebView uses WebView2?
Please let me know how may I test the BlazowWebView content using PlayWright and SDP connection.

**Comment by BmoeCAI** [comment]
Is there any solution here? I'd like to know as well how to accomplish this.
